index.html

```diff
@@ -1,57 +1,329 @@
-                </div>
-
-                <div class="social-icons">
-                    <ul>
-                        <li><a href="https://twitter.com"><img src="images/twitter.png" alt="twitter"></a></li>
-                        <li><a href="https://Linkedin.com"><img src="images/Linkedin.png" alt="Linkedin"></a></li>
-                        <li><a href="https://medium.com"><img src="images/medium.png" alt="medium"></a></li>
-                        <li><a href="https://github.com"><img src="images/github.png" alt="github"></a></li>
-                        <li><a href="https://angellist.com"><img src="images/victory.png" alt="angellist"></a></li>
-                    </ul>
-                </div>
-            </div>
-        </section>
-
-
-
-    </div>
-
-</body>
-
-</html>
\ No newline at end of file
+      <nav class="header">
+          <a class="logo" href="#">My Logo</a>
+          <img src="images/Union.png" alt="">
+      </nav>
+      <section>
+
+          <div class="headline">
+
+              <div class="headline-heading">
+                  <h1>I’m Ahmed Glad to see you!</h1>
+              </div>
+
+              <div class="headline-para">
+                  <p>I’m a software developer! I can help you build a product , feature or website Look through some
+                      of my
+                      work
+                      and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                      contact
+                      me.
+                  </p>
+              </div>
+              <div class="lets-connect">
+                  <h5>LET'S CONNECT</h5>
+              </div>
+
+              <div class="social-icons">
+                  <ul>
+                      <li><a href="https://twitter.com"><img src="images/twitter.png" alt="twitter"></a></li>
+                      <li><a href="https://Linkedin.com"><img src="images/Linkedin.png" alt="Linkedin"></a></li>
+                      <li><a href="https://medium.com"><img src="images/medium.png" alt="medium"></a></li>
+                      <li><a href="https://github.com"><img src="images/github.png" alt="github"></a></li>
+                      <li><a href="https://angellist.com"><img src="images/victory.png" alt="angellist"></a></li>
+                  </ul>
+              </div>
+          </div>
+      </section>
+
+
+
+  </div>
+  <div class="first-page-space"></div>
+
+  <section>
+
+  <div class="snapshot-container">
+
+  <div class="snapshot"><img src="images/Snapshoot Portfolio.png" alt=""></div>
+  <div class="pheading">
+  <h1>Tonic</h1>
+  </div>
+  <div class="date">
+  <ul>
+  <li>Canopy</li>
+  <li><img src="images/dot.png" alt=""></li>
+  <li>Back End Dev</li>
+
+  <li><img src="images/dot.png" alt=""></li>
+  <li>2015</li>
+
+  </ul>
+
+  </div>
+  <div class="summary">
+  <h2>A daily selection of privately personalized reads; no accounts or sign-ups required.</h2>
```

Comment:
- [ ] It is required to use `p` tag for this element instead of the `h2` tag you used. Kindly change the `h2` tag in there to `p` tag.